packages/drawer/Drawer.tsx

```diff
@@ -22,28 +24,41 @@ export class Drawer extends Component<DrawerProps> {
   static get props() {
     return {
       position: prop.string( {
-        attribute: true
+        attribute: {
+          source: true
+        }
       } ),
       visible: prop.boolean( {
-        attribute: true
+        attribute: {
+          source: true
+        }
+      } ),
+      floating: prop.boolean( {
+        attribute: {
+          source: true
+        },
+        initial: true
       } )
     };
   }
 
   position = 'top';
```

Comment:
@Hotell please be so kind and share link to deprecation notice in skate doc/code, all I see is this https://skatejs.gitbooks.io/skatejs/content/docs/api/Component.html#initial

Thanks